### PR TITLE
Enable options on google-analytics init

### PR DESCRIPTION
**Pull request by denneralex**
It’s important for google analytics to be initialized with some options e.g. 'cookieDomain’ that is used to allow testing on localhost and can only be set on create.
This is my suggestion on how to add that.

### CODE REVIEW

**Comment by poteto** [comment]
Restarted the build

File: addon/metrics-adapters/google-analytics.js [hunk 1]
```diff
@@ -18,19 +19,28 @@ export default BaseAdapter.extend({
...
     if (canUseDOM) {
       /* jshint ignore:start */
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
         m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-      window.ga('create', id, 'auto');
       /* jshint ignore:end */
+      
+      if(hasOptions){
```
Comment (poteto) [inline addon/metrics-adapters/google-analytics.js]: Small nitpick, but I prefer spaces:

```js
if (hasOptions) {
```

File: addon/metrics-adapters/google-analytics.js [hunk 2]
```diff
@@ -18,19 +19,28 @@ export default BaseAdapter.extend({
...
   init() {
-    const config = get(this, 'config');
+    const config = copy(get(this, 'config'));
     const { id } = config;
 
     assert(`[ember-metrics] You must pass a valid \`id\` to the ${this.toString()} adapter`, id);
 
+    delete config.id;
+
+    const hasOptions = Ember.isPresent(Object.keys(config));
```
Comment (poteto) [inline addon/metrics-adapters/google-analytics.js]: Can we destructure `isPresent` at the top?

**Comment by denneralex** [comment]
Just did the changes on formatting you asked.

**Pull request merged**